frontend/tests/features/collection.test.ts

```diff
@@ -103,11 +104,21 @@ async function createCollection(
     testCollection.contactName,
     BLUEPRINT_SAFE_TYPE_OPTIONS
   );
+
+  if (tryBadEmail) {
+    // eslint-disable-next-line sonarjs/no-duplicate-string
```

Comment:
IMO it's still worth lifting `#contact-email` as a constant even in test, otherwise our tests would be harder to maintain long term?